test/test_helper.rb

```diff
@@ -2,6 +2,8 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'capybara/rails'
+require 'capybara/poltergeist'
+require 'mocha/mini_test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.